client-misk/src/main/kotlin/app/cash/backfila/client/misk/client/BackfilaClientLoggingSetup.kt

```diff
@@ -0,0 +1,62 @@
+package app.cash.backfila.client.misk.client
+
+import misk.logging.getLogger
+import org.slf4j.MDC
+
+interface BackfilaClientLoggingSetupProvider {
+  fun <T>withBackfillRunLogging(backfillName: String, backfillId: String, wrapped: () -> T) : T
```

Comment:
should always have space after `<T>`
```suggestion
  fun <T> withBackfillRunLogging(backfillName: String, backfillId: String, wrapped: () -> T) : T
```